lib/generators/react_on_rails/templates/lib/tasks/linters.rake

```diff
@@ -0,0 +1,71 @@
+if %w(development test).include? Rails.env
+  namespace :lint do
+    # require "rubocop/rake_task"
+    # require "slim_lint/rake_task"
+    require "scss_lint/rake_task"
```

Comment:
@derencius This was discussed earlier, and @justin808 decided to forgo the exception and just provide a note to the user at the end of the install. I'm still open to further discuss this